### PR TITLE
Add CountString and Whitespace primitive

### LINKED ISSUE #77: Add WhitespaceCount primitive
- Determine how many whitespace characters there are.

### PR DESCRIPTION
**Pull request by tuethan1999**
Fixes #77

### CODE REVIEW

File: nlp_primitives/tests/test_count_string.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+        answers = [2, 3]
+        self.compare(primitive, test_cases, answers)
+
+    def test_nan(self):
+        primitive = self.primitive("the", ignore_case=True,
+                                   ignore_non_alphanumeric=False,
+                                   is_regex=False,
+                                   match_whole_words_only=False)
+        test_cases = pd.Series([np.nan,
```
Comment (thehomebrewnerd) [inline nlp_primitives/tests/test_count_string.py:156]: Might be good to sprinkle in a `None` and `pd.NA` just to confirm there are not problems with those values as well.

File: nlp_primitives/tests/test_whitespace_count.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        primitive_answers = primitive_func(test_cases)
+        return np.testing.assert_array_equal(answers, primitive_answers)
+
+    def test_strings(self):
+        x = pd.Series(['', 'hi im ethan!', 'consecutive.    spaces.', ' spaces-on-ends '])
+        answers = [0, 2, 4, 2]
+        self.compare(self.primitive(), x, answers)
+
+    def test_nan(self):
+        x = pd.Series([np.nan,
```
Comment (thehomebrewnerd) [inline nlp_primitives/tests/test_whitespace_count.py:22]: Maybe add in a `None` and `pd.NA` here as well?

**Review by thehomebrewnerd — COMMENTED** [review]
Add this update to the changelog as well.

**Review by tamargrey — APPROVED** [review]
lgtm

**Pull request merged**